metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -39,7 +39,7 @@
 tmp_dir = Script.get_tmp_dir()
 
 hostname = config['hostname']
-metron_group = config['configurations']['cluster-env']['metron_group']
+hadoop_group = config['configurations']['cluster-env']['user_group']
```

Comment:
Yuck (but thanks for the data).  That's kinda perverse, but since it's in Ambari it's out of our scope to fix.  So I agree with your current usage.  A comment would be nice, since 
a. it's the `hadoop` group not the `users` group; and furthermore
b. its members are the *stack process user ids* not individual users.